python-sdk/nuscenes/utils/splits.py

```diff
@@ -212,6 +214,16 @@ def create_splits_scenes(verbose: bool = False) -> Dict[str, List[str]]:
 
     return scene_splits
 
+def get_prediction_challenge_split(split: str) -> List[str]:
```

Comment:
I'd prefer to move all these changes to a file `nuscenes/eval/predict/splits.py`.

Reply:
Done.